src/Illuminate/Database/Console/ShowCommand.php

```diff
@@ -45,7 +45,8 @@ public function handle(ConnectionResolverInterface $connections)
         $data = [
             'platform' => [
                 'config' => $this->getConfigFromDatabase($database),
-                'name' => $this->getConnectionName($connection, $database),
+                'name' => $connection->getName(),
```

Comment:
We still need pretty names so please revert this one, to `$this->getConnectionName`